PCB-testing/chris-js/pca-test.js

```diff
@@ -12,31 +12,30 @@
  *
  */
 
-// Get dependencies, initialize stuff
-var sleep = require('sleep');
-/*
-	// If the defaults ever need to get changed, uncomment this
-	// and comment out 'var makePwm = require('adafruit-pca9685');'
-	// Note that the contents of this object are the defaults
+// Should cycle a thruster on and off.
 
-var makePwm =  {
-    "freq": "50",				// frequency of the device
-    "correctionFactor": "1.0",	// correction factor - fine tune the frequency
-    "address": "0x40",			// i2c bus address
-    "device": '/dev/i2c-1',		// device name
-    "debug": <null>				// adds some debugging methods if set
-}
+var i2cBus = require("i2c-bus");
+var Pca9685Driver = require("pca9685").Pca9685Driver;
 
-*/
-var makePwm = require('adafruit-pca9685');
-var pwm = makePwm();
+var options = {
+     i2c: i2cBus.openSync(1),
+     address: 0x40,
+     frequency: 50,
+     debug: false
+};
 
-// Tell PCA to do the things, print to console
+// Loops through turning on and off
 while(true) {
```

Comment:
You don't have to put a loop here, intervals run 4ever (or until you call `clearInterval` on them).